<launch>


     <node pkg="annotated_planar_patch_map" type="annotated_map_snapshotter"  name="am_snapshotter" respawn="false" output="screen">
        <remap from="build_annotated_map" to="/blank_map_server/build_map" />
        <remap from="full_map" to="/patch_maps/map_l1" />
	<param name="snapshot_mode" value="rolling"/>
	<param name="snapshot_timeout" value="10.0"/>
     </node>


     <node pkg="annotated_planar_patch_map" type="annotated_map_snapshotter"  name="am_snapshotter_l2" respawn="false" output="screen">
        <remap from="build_annotated_map" to="/blank_map_server_l2/build_map" />
        <remap from="full_map" to="/patch_maps/map_l2" />
   	<param name="snapshot_mode" value="all"/>
	<param name="snapshot_timeout" value="30.0"/>
     </node>




</launch>
